README.markdown

```diff
@@ -273,15 +286,57 @@ As a character I want to be able to have items that enhance my capabilities so t
 
 #### Samples
 
-As a character I want to be able to wear a ring of protection so that I can be protected from attack
+> As a character I want to be able to wear a ring of protection so that I can be protected from attack
 
   - adds +2 to armor class
 
-As a character I want to be able to war a belt of giant strength so that I can be even stronger in combat
+> As a character I want to be able to war a belt of giant strength so that I can be even stronger in combat
 
   - add +4 to Strength Score
+  
+## Iteration 5 - Improvements to existing features
```

Comment:
I like these additions, but just a fair warning... Most pairs do not complete Iteration 4 (most only making it to the end of Iteration 2 or the beginning of Iteration 3) in an 8 hour period, so don't expect many people to make it to this part before the end of Clean Code Days. :-)